cypress/support/step_definitions/organisation/details/details.ts

```diff
@@ -0,0 +1,43 @@
+import { Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+import { getOrganisationId } from "../../../utils";
+
+Given("an organization exists", () => {
+  cy.createOrganisation(getOrganisationId());
+});
+
+Given("a source exists in the organisation", () => {
+  cy.createOrgSource(undefined, getOrganisationId());
+});
+
+Given("the user is on the organisation details page", () =>
+  cy.visit(`/orgs/${getOrganisationId()}/`)
+);
+
+Then("the organisation name should be displayed", () => {
+  cy.findByText("Details").should("be.visible");
+  cy.findByText("Test Organisation", { selector: "span" }).should("be.visible");
+});
+
+Then("the organisation sources should be displayed", () => {
+  cy.findByText("Sources", { selector: "legend" }).should("be.visible");
+});
+
+Then("the organisation members should be displayed", () => {
+  cy.findByText("Members", { selector: "legend" }).should("be.visible");
+});
+
+Then("the organisation dictionaries should be displayed", () => {
+  cy.findByText("Dictionaries", { selector: "legend" }).should("be.visible");
+});
+
+Then("the user should see the organisation source", () => {
+  cy.findByText("Test Org Source", { selector: "li" }).should("be.visible");
```

Comment:
It's the `<a>` element that has the relevant text (I suppose something like `li > *` might work too)

```suggestion
  cy.findByText("Test Org Source", { selector: "a" }).should("be.visible");
```

Reply:
@hadijahkyampeire Did you see this comment? I believe this is related to why the test is failing. In particular, I think the implementation of `findByText` is such that it won't find "Test Org Source" as the text for the `li` element (see [the actual implementation](https://github.com/testing-library/dom-testing-library/blob/45830f580ced3ae5337af9ace4a9bd65067dfe5c/src/queries/text.ts#L13)).

In HTML, the element we're targetting looks like this:

```html
<ul>
  <li>
    <a href="...">Test Org Source</a>
  </li>
</ul>
```